apps/studio/components/interfaces/Auth/ThirdPartyAuthForm/IntegrationCard.tsx

```diff
@@ -0,0 +1,214 @@
+        <div className="py-1">
+          <Image src={getIntegrationTypeIcon(type)} width={21} height={21} alt={`${type} icon`} />
+        </div>
+        <div className="flex flex-col flex-0 overflow-y-auto w-full gap-y-4">
+          <div className="text-sm flex flex-col">
+            <span className="text-foreground">{getIntegrationTypeLabel(type)}</span>
+            <div className="text-foreground-lighter">{getIntegrationTypeDescription(type)}</div>
+          </div>
+
+          <IntegrationTypeContent type={type} integration={integration} />
+          <div>
+            {/* TODO: this should be a configure integration where it would show the sheet and the user can disable or delete the integration
+              but there's no "edit integration" endpoing for now. */}
+            <Button type="danger" disabled={!canUpdateConfig} onClick={() => onSelect()}>
+              Delete integration
+            </Button>
+          </div>
+        </div>
+        <div className="flex-1">
+          {true ? (
+            <Badge className="space-x-1" size="large" variant="brand">
+              <div className="h-3.5 w-3.5 bg-brand rounded-full flex justify-center items-center">
+                <Check className="h-2 w-2 text-background-overlay" strokeWidth={6} />
+              </div>
+              <span>Enabled</span>
+            </Badge>
+          ) : (
+            <Badge variant="warning" size="large">
+              <span>Disabled</span>
+            </Badge>
+          )}
+        </div>
+      </div>
+      {type === 'firebase' ? (
+        <div className="bg-alternative border overflow-hidden shadow pl-11 pr-4 py-5 flex flex-row space-x-4">
+          <div className="bg-warning h-5 w-5 text-black flex items-center justify-center rounded-md">
+            <ExclamationCircleIcon className="h-4 w-4 text-black" />
+          </div>
+          <div className="text-sm space-y-4">
+            <div className="space-y-1">
+              <p className="text-foreground">Firebase provider uses a custom RLS policy</p>
+              <p className="text-foreground-light">
+                Be aware that this provider requires an RLS policy to correctly work.
+              </p>
+            </div>
+
+            <Dialog>
+              <DialogTrigger asChild>
+                <Button type="warning" size="tiny">
+                  Insert policy
+                </Button>
+              </DialogTrigger>
+              <DialogContent size="xlarge">
+                <AddRLSPolicyForFirebaseDialog
+                  projectRef={projectRef!}
+                  firebaseProjectId={
+                    integration.oidc_issuer_url?.replace('https://securetoken.google.com/', '') ||
+                    ''
+                  }
+                />
+              </DialogContent>
+            </Dialog>
+          </div>
+        </div>
+      ) : null}
+    </>
+  )
+}
```

Comment:
Just adding some UI feedback here:

Should we just put the Project ID / Domain name / User pool ID in the header? helps to differentiate easier if there are multiple of the same

![image](https://github.com/user-attachments/assets/664158fb-043b-4dd3-8fb7-8706a8888605)

Reply:
imo these cards feel like they're taking up too much space than they need to, especially for firebase where its really wordy

![image](https://github.com/user-attachments/assets/cbbe49a1-51b1-40e2-a8d9-524da1c260b8)

Reply:
We could take some inspiration (and also consistent UI) from the auth providers page? Use collapsibles, hide all these information in there

![image](https://github.com/user-attachments/assets/6b1349ed-dfe8-479c-b3c6-7df452aded06)

Reply:
For callouts that involve a subsequent action - i'd rather we make them more obvious with an `<Alert_Shadcn />` tbh, otherwise it's really easy to overlook. In the case of Firebase here - maybe we show the Alert _only if_ the user has not created any RLS policy for any table? and if they've already created one, then we probably don't need to make it too obvious with an alert, maybe something less "CTA"ish by avoiding words like "You need to..."

![image](https://github.com/user-attachments/assets/0442d00c-02af-4728-9eb2-520528e35480)

Reply:
For Auth0 here - "Additional set up may be required", is it possible to state exactly what set up is required? Ideal UX we should aim for is to reduce the amount of thinking the user has to do. In this I'm not sure where to look out for immediately when i open the docs, cause i'm looking for a header that resembles "Additional set up"

![image](https://github.com/user-attachments/assets/74fcca42-ffad-4b8b-bf94-a6c3eb976394)

Reply:
I'm not too sure about Cognito cause i'm having trouble setting up an integration

but cc @MildTomato as well for this thread on design feedback 🙏

Reply:
This UI has gone a bit off what was originally pitched and it's now getting designed by committee. Just go ahead and get it out and I'll come back to it later to help get all Auth settings improved as a whole. I think trying to introduce a new UI pattern here was a mistake and now it looks pretty bad compared the original designs.